website/src/blog/2023-08-15-fusion/2023-08-15-fusion.md

```diff
@@ -0,0 +1,592 @@
+
+When GraphQL was introduced, front-end engineers where the first to glimpse the power of it and started wrapping their REST services with GraphQL. This made data fetching more efficient by aggregating data calls close to downstream services and rendered the data more accessible. The straightforward, human-understandable schema made it easier to trace relations and reason about data and its connections in an entirely new way.
+
+GraphQL offered a way to model an interface to our core business domain that often diverged from the technical realities of the REST, gRPC, or other APIs behind it. It eliminated the complexity of knowing which micro-service would provide the necessary data or mutations for a particular use case. While micro-service or domain-service architectures provided technical means to scale more efficiently and align with organizational needs, GraphQL introduced simplicity with its unified schema approach.
+
+From the outset, GraphQL server developers were challenged to find ways to simplify distributed GraphQL setups. Over time, we've witnessed the evolution of various methods, from schema stitching techniques to federated solutions like Apollo Federation. However, many of these either restrict users within a single-vendor ecosystem or, on the other end of the spectrum, are too rudimentary to cater to sophisticated enterprise requirements.
+
+## Expectations
+
+Our belief is that distributed GraphQL services — or composite GraphQL services — should not only be straightforward to set up but also seamlessly integrate with the diverse range of CI/CD tools, schema registries, composition utilities, and gateways that enterprises might prefer. The current landscape should not dictate the choice of tools but provide flexibility.
+
+Up to this point, the GraphQL landscape has lacked an open specification tailored for distributed setups – a framework designed from the ground up for extensibility and integration with diverse toolchains. We envisioned a platform where tools from various vendors could effortlessly work in tandem, ensuring that developers and enterprises never feel constrained by their technical choices.
+
+## Lets share and compete
+
+Late last year ChilliCream and The Guild met in Paris and discussed their approaches twoards distributed GraphQL. It became very clear that both companies were solving similar problems and we decided to join forces on this project. ChilliCream would provide the initial work on the Fusion spec and implementation while The Guild would start specifying their work on Mash and Open API and help shape the initial Fusion spec.
+
+Today, we are thrilled to unveil GraphQL-Fusion an open specification under the MIT license. This initiative empowers everyone to craft tools and solutions centered around distributed GraphQL services. Complementing this announcement, we're also introducing Hot Chocolate Fusion, an early implementation of the GraphQL-Fusion spec draft.
```

Comment:
```suggestion
Today, we are thrilled to unveil GraphQL-Fusion an open specification under the **MIT license**. This initiative empowers everyone to craft tools and solutions centered around distributed GraphQL services. Complementing this announcement, we're also introducing Hot Chocolate Fusion, an early implementation of the GraphQL-Fusion spec draft.
```

Reply:
Should you also mention here the working group?

Reply:
and are you ok with adding something like "And The Guild commitment to integrate Fusion into their existing gateway solutions" or something like that?